<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生管理系统-Python课程设计</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>570</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_1_info_query">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>个人信息查询</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>470</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string>个人信息一览</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>431</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_1" rowstretch="0,0,0,0,0,0" columnstretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>学  院：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>姓  名：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>年  龄：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>班  级：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>性  别：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>学  号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_1_sid">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2_sname">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_3_ssex">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_4_sage">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_5_sdept">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_6_sclass">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_Sid">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>按学号查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Sname">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>280</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>按姓名查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_submit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>440</y>
        <width>130</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>提交更改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>440</y>
        <width>130</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>录入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2_valiable_course">
     <attribute name="title">
      <string>可用课程查询</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>选择查询</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
       <item>
        <property name="text">
         <string>请选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按课程名称</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按教师姓名</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_val">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_select">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>761</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>可用课程</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_course">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>721</width>
         <height>251</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_commit">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>选课</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>541</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>学生ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Sid">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>课程ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pid">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3_score_query">
     <attribute name="title">
      <string>教学班成绩查询</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>761</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>班级均分</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>721</width>
         <height>311</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_select">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>选择查询</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLength</enum>
       </property>
       <item>
        <property name="text">
         <string>请选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按课程名称</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按教师姓名</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_val_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_select_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4_import">
     <attribute name="title">
      <string>数据导入</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>661</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>源文件</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>591</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_path">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>选择表格</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>学生信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教师信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>班级信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>选课信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>课程信息</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教学计划信息</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>160</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_browse">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_import">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>896</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
